CSVEnumerable/Program.cs

```diff
@@ -0,0 +1,9 @@
+﻿using CSVEnumerable;
```

Comment:
Could you please investigate this code fragment? It means `IEnumerator IEnumerable.GetEnumerator()` doesn't support all options of the iterator pattorn in .NET. Is it possible to fix it somehow?
```
var csvItems = new CsvEnumerable<Person>("document.csv");

var enumerator = csvItems.GetEnumerator();

while (enumerator.MoveNext())
{
    Console.WriteLine($"{enumerator.Current.Name}, {enumerator.Current.Id}");
}

enumerator.Reset();

while (enumerator.MoveNext())
{
    Console.WriteLine($"{enumerator.Current.Name}, {enumerator.Current.Id}");
}
```

Reply:
Solution is implemented